Majasdarbs/10-patstavigais-darbs/script.js

```diff
@@ -0,0 +1,127 @@
+window.addEventListener('DOMContentLoaded', () => {
```

Comment:
Instead of this you can place script file in the bottom of html body, or load it with defer attribute